mqt/bench/tests/test_all.py

```diff
@@ -255,36 +255,335 @@ def test_routing():
 @pytest.mark.parametrize(
     "benchmark_name, level, circuit_size, benchmark_instance_name, compiler, compiler_settings, gate_set_name, device_name,",
     [
-        ("dj", "alg", 5, None, "qiskit", 1, None, None),
-        ("wstate", 0, 6, None, "qiskit", 2, None, None),
-        ("wstate", 0, 6, None, "tket", True, None, None),
-        ("wstate", 0, 6, None, "tket", False, None, None),
-        ("ghz", "indep", 5, None, "qiskit", 3, None, None),
+        (
+            "dj",
+            "alg",
+            5,
+            None,
+            None,
+            None,
+            None,
+            None,
+        ),
+        (
+            "wstate",
+            0,
+            6,
+            None,
+            None,
+            None,
+            None,
+            None,
+        ),
+        (
+            "wstate",
+            0,
+            6,
+            None,
+            None,
+            None,
+            None,
+            None,
+        ),
+        (
+            "wstate",
+            0,
+            6,
+            None,
+            None,
+            None,
+            None,
+            None,
+        ),
```

Comment:
These are just copy-pasted here and do not reflect the tests that were conducted before. Please also check all tests below. Some of them contain copy-paste errors as well.

Reply:
done